server/src/main/java/org/elasticsearch/search/aggregations/metrics/TDigestState.java

```diff
@@ -47,23 +49,35 @@ static Type valueForHighAccuracy() {
 
     private final Type type;
 
+    // TODO: DELETE ME
```

Comment:
Is this still used?

Reply:
I saw the comment.. Nit: let's be specific about when this can be deleted (e.g. once there are no more references in tests?)